<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfacciaModificaProfilo</class>
 <widget class="QWidget" name="InterfacciaModificaProfilo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(113, 113, 113);
}

QPushButton
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0.507177 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 127, 255));
	font: bold 16pt &quot;Kinnari&quot;;
	color: rgb(255, 170, 0);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

QPushButton:hover
{
	background-color:qlineargradient(spread:pad, x1:0.474, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:0.507177 rgba(255, 170, 0, 255)); 
	color: rgb(0, 0, 0);
}


QPushButton:hover:!pressed
{
	background-color:qlineargradient(spread:pad, x1:0.244, y1:1, x2:0.837, y2:0.227273, stop:0 rgba(255, 255, 127, 255), stop:0.507177 rgba(0, 0, 0, 255));
	color: rgb(255, 170, 0);
}

#ModificaProfiloLabel
{
	font: bold italic 32pt &quot;Kinnari&quot;;
	color: rgb(255, 85, 0);
	background-color:rgb(113, 113, 113);
}

#DatiPersonaliLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#ContattiLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#CambiaPasswordLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#LingueLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

#EsperienzeLabel
{
	font: bold 18pt &quot;Kinnari&quot;;
	color:rgb(255, 170, 0);
	background-color:rgb(113, 113, 113);
}

QLabel
{
	font: bold 14pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color:rgb(57, 57, 57);
}

QLineEdit
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QTextEdit
{
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QDateEdit
{ 
	font:bold 12pt &quot;Kinnari&quot;;
	color:rgb(255, 255, 127);
	background-color: rgb(35, 35, 35);
	selection-color: rgb(0, 0, 0);
	selection-background-color: rgb(255, 170, 0);
	border-style: solid;
	border: 2px solid rgb(255, 170, 0);
	border-radius: 7;
}

QGroupBox
{
	background-color:rgb(57, 57, 57);
	border-width: 2px;
	border-color:rgb(255, 85, 0);
	border-style: solid;
	border-radius: 7;
}

#line2
{
	background-color:rgb(57, 57, 57);
}</string>
  </property>
  <widget class="QLabel" name="ModificaProfiloLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>300</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Modifica Profilo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>700</width>
     <height>10</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>660</width>
     <height>160</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="NomeLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="CognomeLineEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="DataNascitaDateEdit"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="LuogoNascitaLineEdit"/>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="IndirizzoLineEdit"/>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="CittaLineEdit"/>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="ProvinciaLineEdit"/>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="TitoloStudioLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="CognomeLabel">
      <property name="text">
       <string>Cognome</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Provincia</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="NomeLabel">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="TitoloStudioLabel">
      <property name="text">
       <string>Titolo Di Studio</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="LuogoNascitaLabel">
      <property name="text">
       <string>Luogo Nascita</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="DataNascitaLabel">
      <property name="text">
       <string>Data Nascita</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="CittaLabel">
      <property name="text">
       <string>Citt√†</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="IndirizzoLabel">
      <property name="text">
       <string>Indirizzo</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="4">
     <widget class="Line" name="line2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>295</y>
     <width>280</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Telefono</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="TelefonoLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="EmailLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>295</y>
     <width>350</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="NuovaPasswordLabel">
      <property name="text">
       <string>Nuova Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="NuovaPasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ConfermaPasswordLabel">
      <property name="text">
       <string>Conferma Password</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="ConfermaPasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="LingueTextEdit">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>420</y>
     <width>260</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="EsperienzeLavorativeTextEdit">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>420</y>
     <width>260</width>
     <height>200</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfermaModificheButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>640</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AnnullaButton">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>640</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QLabel" name="DatiPersonaliLabel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Dati Personali</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ContattiLabel">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>265</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Contatti</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="CambiaPasswordLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>265</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cambia Password</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LingueLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Lingue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EsperienzeLabel">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>390</y>
     <width>220</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Esperienze Lavorative</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
